`TestLogicalOverflow` failed

## Flaky Test

### Which jobs are failing
```
--- FAIL: TestLogicalOverflow (3.74s)
    global_tso_test.go:168: 
        	Error Trace:	/home/runner/work/pd/pd/tests/server/tso/global_tso_test.go:168
        	            				/home/runner/work/pd/pd/tests/server/tso/global_tso_test.go:176
        	Error:      	"25.822158ms" is not less than "25ms"
        	Test:       	TestLogicalOverflow
```

### CI link
https://github.com/tikv/pd/actions/runs/6033191113/job/16369522346?pr=7002

### Reason for failure (if possible)


### Anything else

--- a/tests/server/tso/global_tso_test.go
+++ b/tests/server/tso/global_tso_test.go
@@ -165,7 +165,7 @@ func TestLogicalOverflow(t *testing.T) {
 			re.NoError(err)
 			if i == 1 {
 				// the 2nd request may (but not must) overflow, as max logical interval is 262144
-				re.Less(time.Since(begin), updateInterval+20*time.Millisecond) // additional 20ms for gRPC latency
+				re.Less(time.Since(begin), updateInterval+50*time.Millisecond) // additional 50ms for gRPC latency
 			}
 		}
 		// the 3rd request must overflow

